### ISSUE TITLE
finishing warehouse crashed game and now cant get back in save

### ISSUE THREAD

**Issue by Brainslime1** [ldtteam/minecolonies#2487]
version 1.12.2-0.8.7582

when my builder finished a warehouse it crashed my game and now I cant get back in 

dropbox=https://www.dropbox.com/s/t80i1ogwpmiy2b2/crash-2018-05-27_09.06.43-server.txt?dl=0

**Comment by Raycoms** [comment]
Can you try the newest alpha? And, it seems like there is a mod which adds
ores to the game causes this since they don't have their ores implemented
accordingly.

2018-05-27 14:59 GMT+01:00 dylan <notifications@github.com>:

**Comment by Brainslime1** [comment]
it still didn't work

https://www.dropbox.com/s/w2u5988fzsj10d6/crash-2018-05-27_09.53.43-server.txt?dl=0

**Comment by Nightenom** [comment]
What about this?
```
Ender IO: Found the following problem(s) with your installation (That does NOT mean that Ender IO caused the crash or was involved in it in any way. We add this information to help finding common problems, not as an invitation to post any crash you encounter to Ender IO's issue tracker. Always check the stack trace above to see which mod is most likely failing.):
                  * Optifine is installed. This is NOT supported.
                 This may (look up the meaning of 'may' in the dictionary if you're not sure what it means) have caused the error. Try reproducing the crash WITHOUT this/these mod(s) before reporting it.
```

**Comment by Brainslime1** [comment]
i'll try it but the only mod I have that I know has a problem with optifine the gun mod and that's if a certain setting is on

**Comment by Brainslime1** [comment]
i tried without it and it still didn't work

this is without optifine 
https://www.dropbox.com/s/10xxnqccoxs9452/crash-2018-05-27_16.40.05-server.txt?dl=0

**Comment by PoroUsedSnax** [comment]
Have you tried using a new world since you've updated? It's possible there could be an issue with the world.

That's presuming it's a world you're having issues with, not actually loading the game.

**Comment by Brainslime1** [comment]
I'm able to make a new save. the problem happened right as the game was like hey your builder finished this building

**Comment by marchermans** [comment]
This Crash has nothing todo with minecolonies. I see no reference in the Crash Report. Just because it seemed to happen directly after does not make it minecolonies fault. However can you reproduce it in all worlds?

**Comment by Brainslime1** [comment]
i built a warehouse on a new save and it didn't crash so

**Comment by Ragnarok96** [comment]
@Brainslime1 If anything that to me would mean even more so that the crashes you were experiencing before were not related to minecolonies especially as like said @OrionDevelopment said minecolonies is not specifically mentioned.

Additionally, after reviewing your crash report I believe this may be to do with thaumcraft especially if you have recently updated it, my server would always crash with that exact message after I updated thaumcraft from b13 to b14 or b15. The only fix was to revert to B13, so it was good I did a backup before updating all of the mods in my scenario.

It appears you are using thaumcraft beta 14 and therefore this may be causing your issues - there are a lot of crash bugs with b14 & 15.

See here:
https://github.com/Azanor/thaumcraft-beta/issues/841
